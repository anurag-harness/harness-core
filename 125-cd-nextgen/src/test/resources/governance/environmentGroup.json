{
  "envGroupRef": "env_group_1",
  "metadata": {
    "parallel": true
  },
  "deployToAll": false,
  "environments": [
    {
      "environmentRef": "my_environment",
      "infrastructureDefinitions": [
        {
          "identifier": "my_infra_with_ns_runtime",
          "inputs": {
            "identifier": "my_infra_with_ns_runtime",
            "type": "KubernetesDirect",
            "spec": {
              "namespace": "harness",
              "releaseName": "dev-0001"
            }
          }
        }, {
          "identifier": "some-<+expression>-with_should_be_ignored"
        },
        {
          "identifier": "my_infra"
        }
      ]
    },
    {
      "environmentRef": "non_existent_environment",
      "infrastructureDefinitions": [
        {
          "identifier": "some_infra"
        }
      ]
    }
  ]
}