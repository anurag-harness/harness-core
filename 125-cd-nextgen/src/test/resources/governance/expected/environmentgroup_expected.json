{
  "identifier" : "xyz",
  "environments" : [ {
    "identifier" : "my_env",
    "name" : "my env",
    "type" : "Production",
    "orgIdentifier" : "org",
    "projectIdentifier" : "proj",
    "infrastructures" : [ {
      "type" : "KubernetesDirect",
      "spec" : {
        "namespace" : "default",
        "releaseName" : "release_name",
        "connector" : {
          "name" : "k8s",
          "identifier" : "k8s",
          "type" : "K8sCluster",
          "spec" : {
            "key" : "value"
          }
        }
      }
    } ]
  } ],
  "name" : "wave1",
  "envGroupRef" : "account.wave1"
}